src/twisted/mail/test/test_imap.py

```diff
@@ -7105,9 +7105,9 @@ def cbCopy(results):
 class TLSTests(IMAP4HelperMixin, TestCase):
     serverCTX = None
     clientCTX = None
-    if ServerTLSContext:
+    if ServerTLSContext:  # type: ignore[truthy-function]
```

Comment:
would something like this work ?
```suggestion
    if ServerTLSContext is not None:
```